text/0001-compsci-bois-almanac.md

```diff
@@ -0,0 +1,115 @@
+- Project / Feature Name: `compsci-bois-almanac`
+- Start Date: 2023-08-17
+- RFC PR: [fogo-sh/rfcs#1](https://github.com/fogo-sh/rfcs/pull/1)
+
+# Summary
+
+[summary]: #summary
+
+A collection of markdown files about different events / things / memes / etc. in the Compsci-Boi-verse, accessible over
+http(s) and gemini, privately.
+
+Images and videos will be supported, but it will be mostly text.
+
+The initial content can be populated from a XML export of the existing Compsci Bois Wiki.
+
+Backlinks will be supported via \[\[this syntax\]\].
+
+After the initial version is released, more syntax can be added to increase more wiki-like functionality, like
+categories / dates / other enriching metadata, along then with pages that can act as indexes that utilize the metadata
+(like a all categories page, or an all posts about events in 2023).
+
+# Motivation
+
+[motivation]: #motivation
+
+Having a location to document different things about our group, from a nostalgic / historical perspective, would be
+nice.
+
+Hopefully, due to the enjoyment of consuming even the existing content of this resource that comes from the existing
+wiki in this new more accessible to programmer format will encourage more people to contribute to it.
```

Comment:
I think general awareness of the wiki and account access is probably a bigger barrier right now.  Markdown would be neat, but I for one forgot the wiki ever actually _got made_ and wasn't just something that was talked about in theory. If we had a Discord bot that could hand out accounts for the wiki that would probably be the least frictional path out of all of them. The Wiki markup isn't very complicated at all and allows WYSIWYG editing which would probably be faster for some.